docs/docs/installation/configuration/assets/argo-app.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: keptn
+  namespace: argocd
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io # enabling cascading deletion
+spec:
+  project: default
+  source:
+    repoURL: 'https://github.com/keptn/lifecycle-toolkit-charts' # repository containing helm charts
+    path: charts/keptn # path to helm charts inside repository
+    targetRevision: HEAD
```

should this really be the actual git repo url and not the helm chart repo? I think it's the latter:
```suggestion
    repoURL: 'https://charts.lifecycle.keptn.sh' # repository containing helm charts
```

---

this does not work

---

the repoURL needs to be a url to git repository

---

there should be chart: keptn present then in the application, let me try it out

---

adapted